src/github.com/getlantern/flashlight/client/fronted.go

```diff
@@ -71,22 +77,34 @@ func (s *FrontedServerInfo) dialer(masqueradeSets map[string][]*fronted.Masquera
 		MaxMasquerades:     s.MaxMasquerades,
 		RootCAs:            globals.TrustedCAs,
 	})
-	masqueradeQualifier := ""
+
+	// Are we using a masquerade set?
+	var masqueradeQualifier string
 	if s.MasqueradeSet != "" {
 		masqueradeQualifier = fmt.Sprintf(" using masquerade set %s", s.MasqueradeSet)
 	}
-	return fd, &balancer.Dialer{
-		Label:  fmt.Sprintf("fronted proxy at %s:%d%s", s.Host, s.Port, masqueradeQualifier),
-		Weight: s.Weight,
-		QOS:    s.QOS,
-		Dial:   fd.Dial,
+
+	// Is this a trusted proxy that we could use for HTTP traffic?
+	var trusted string
+	if s.Trusted {
+		trusted = "(trusted) "
+	}
```

Comment:
As said above, maybe it would be clearer to explicitly state "untrusted " vs "trusted " instead of assuming the former by default.